### PR TITLE
v11.1.0

### PR DESCRIPTION
**Pull request by h-vetinari**
Trying to fix the errors from #130. ~Might be slightly over-engineered, but I didn't want to touch the internals of `make install`.~

Closes #130

### CODE REVIEW

**Comment by isuruf** [comment]
In this case, we do want the minor version. `libclang-cpp11.1` instead of `libclang-cpp11`

**Comment by h-vetinari** [comment]
@isuruf

PTAL

**Review by isuruf — COMMENTED** [review]
We also need to fix the name in Line 112 `libclang-cpp{{ major_version }}` to `libclang-cpp{{ minor_aware_ext }}` and the uses of it.

**Comment by h-vetinari** [comment]
OK, can do.

Just one question, on OSX, we have the following:
```
-- Installing: /Users/runner/miniforge3/conda-bld/[...]/lib/libclang.11.1.dylib
-- Installing: /Users/runner/miniforge3/conda-bld/[...]/lib/libclang.11.dylib
-- Installing: /Users/runner/miniforge3/conda-bld/[...]/lib/libclang.dylib
```

Should `libclang.11.dylib` be removed?

**Comment by isuruf** [comment]
On OSX, the patch at https://github.com/conda-forge/clangdev-feedstock/blob/master/recipe/patches/0001-Set-VERSION-in-osx-as-well.patch#L20 should be updated to set `SOVERSION` as well.

**Comment by h-vetinari** [comment]
@isuruf
Thanks for the instructions, this should be ready.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -186,10 +192,15 @@ outputs:
...
+        - test -f "$PREFIX/lib/libclang.so.{{ minor_aware_ext }}"     # [linux]
+        {% if minor_aware_ext != major_version %}
+        - test ! -f "$PREFIX/lib/libclang.so.{{ major_version }}"     # [linux]
+        - test ! -f "$PREFIX/lib/libclang.{{ major_version }}.dylib"  # [osx]
+        {% endif %}
+        - test -f "$PREFIX/lib/libclang.dylib"                        # [osx]
+        - test -f "$PREFIX/lib/libclang.{{ minor_aware_ext }}.dylib"  # [osx]
+        - if not exist %LIBRARY_BIN%\\libclang.dll exit 1             # [win]
 
   - name: clang-{{ major_version }}
```
Comment (h-vetinari) [inline recipe/meta.yaml:205]: Maybe worth a comment that this package hasn't changed name - I haven't been able to follow which packages should reflect the minor version and which shouldn't; just double-checking.

**Pull request merged**